README.md

```diff
@@ -11,9 +11,13 @@ Use this tutorial: http://www.howtogeek.com/249966/how-to-install-and-use-the-li
 1. Install the latest Python 2.7: https://www.python.org/downloads/release/python-2712/
 2. Go into your unix system and install SciPy: https://www.scipy.org/install.html
   1. Note that installation might be different for different systems
+  2. For Ubuntu, try: sudo apt-get install python-numpy python-scipy python-matplotlib ipython ipython-notebook python-pandas python-sympy python-nose
 3. Install SciKit-Learn using pip: http://scikit-learn.org/stable/install.html
 
 ###### Miscellaneous
 - We are using https://www.zenhub.com/ for project management.
 - We are using https://studio.azureml.net/ for initial data analysis and some machine learning.
 - Our Google Drive with our other documents can be viewed here: https://drive.google.com/drive/folders/0B_C34Fpc9Zf_TDlRUDhlZ0c3WVE?usp=sharing
+
+###### Coding Standards (Please Call Us On It)
```

Comment:
I'll wait until we have more content to have in a file like that, but it'll get done next iteration